### DISCUSSION TITLE
filteringText & validatingText

### DISCUSSION THREAD

**Discussion by gputignano** [SBoudrias/Inquirer.js#1193] [General]
I can't find any documentation about filteringText & validatingText as in the example filter-validate-progress.js.

**Comment by SBoudrias** [comment]
Hi, those are documented here https://github.com/SBoudrias/Inquirer.js/tree/master#objects (although I agree this might not be the clearest place to be.) I'm open to suggestion to make those clearer.

**Reply by gputignano** [reply]
I can't find filteringText and validatingText reference at that page.

**Reply by SBoudrias** [reply]
Oh... I now see. To be honest I didn't even remember we had this feature 🤦🏻 

Feel free to send a PR documenting those options!
